### PR TITLE
added data resiliency integration

### PR DESCRIPTION
**Pull request by afreen23**
[kubevirt/web-ui-components/pull/326](https://github.com/kubevirt/web-ui-components/pull/326)

### CODE REVIEW

File: frontend/public/storage/components/storage-overview/storage-overview.jsx [hunk 1]
```diff
@@ -19,6 +19,8 @@ import { coFetchJSON } from '../../../co-fetch';
 const REFRESH_TIMEOUT = 5000;
 
 const CEPH_STATUS = 'ceph_health_status';
+const CEPH_PG_ACTIVE = 'ceph_pg_active';
```
Comment (suomiy) [inline frontend/public/storage/components/storage-overview/storage-overview.jsx]: can you add a `QUERY` suffix to these variables?

File: frontend/public/storage/components/storage-overview/storage-overview.jsx [hunk 2]
```diff
@@ -82,6 +86,15 @@ export class StorageOverview extends React.Component {
       diskStats: {
         ...state.diskStats,
         [key]: response,
+      }
+    }));
+  }
+
+  _setDataResiliencyData(key, response) {
```
Comment (suomiy) [inline frontend/public/storage/components/storage-overview/storage-overview.jsx]: the code for `_setCephDiskStats`, `_setDataResiliencyData`, etc. is very similar. Can you please make it generic?

File: frontend/public/storage/components/storage-overview/storage-overview.jsx [hunk 3]
```diff
@@ -39,7 +42,7 @@ const resourceMap = {
   },
 };
 
-const getPrometheusBaseURL = () => window.SERVER_FLAGS.prometheusBaseURL;
+const getPrometheusBaseURL = () => 'https://prometheus-k8s-openshift-monitoring.apps.uchapaga.devcluster.openshift.com';
```
Comment (suomiy) [inline frontend/public/storage/components/storage-overview/storage-overview.jsx]: this shouldn't be here :)

Comment (afreen23) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: done

File: frontend/public/storage/components/storage-overview/storage-overview.jsx (hunk 2 above)
Comment (afreen23) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: done

File: frontend/public/storage/components/storage-overview/storage-overview.jsx (hunk 1 above)
Comment (afreen23) [reply frontend/public/storage/components/storage-overview/storage-overview.jsx]: done

**Pull request merged**